libs/application/template-api-modules/src/lib/modules/templates/parental-leave/parental-leave.utils.ts

```diff
@@ -517,6 +509,29 @@ export const checkActionName = (
       params === 'employer'
     return params
   }
+
```

Comment:
do we need to setActionName in the template and just use the findActionName logic in the checkActionName function?

Reply:
and if so, rename the checkActionName function to something more descriptive..

Reply:
We need to use setActionName to keep bookkeeping of what actionName has been selected when sending application